.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "esp-idf"]
+	path = third_party/esp-idf/esp-idf
```

Comment:
Let's make this `third_party/esp-idf` and avoid the double nesting.

Reply:
Hmm, toyed around with git rm's automatic .gitmodule update feature. It must have fooled me somehow. Fixed.